disco_aws_automation/disco_elastigroup.py

```diff
@@ -88,7 +88,7 @@ def get_existing_groups(self, hostclass=None, group_name=None):
                 'launch_config_name': None,
                 'termination_policies': [],
                 'vpc_zone_identifier': ','.join(
-                    zone['subnetId'] for zone in group['compute']['availabilityZones']
+                    subnet for subnets in group['compute']['availabilityZones'] for subnet in subnets['subnetIds']
```

Comment:
Could improve the naming of the variables:
```suggestion
                    subnet for zone in group['compute']['availabilityZones'] for subnet in zone['subnetIds']
```

Reply:
Also line is too long.